files/templates/alertmanager/alertmanager.yaml

```diff
@@ -664,12 +664,6 @@ inhibit_rules:
   - cancel_if_nodes_down=true
   equal: [cluster_id]
 
-- source_matchers:
-  - scrape_timeout=true
-  target_matchers:
-  - cancel_if_scrape_timeout=true
-  equal: [cluster_id, instance]
-
```

Comment:
This change is not related to the refactoring here and has potential to break, please move this to a different PR

Reply:
Yes it was done in another PR, but this branch was old :(